assets/app/views/builds.html

```diff
@@ -1,20 +1,34 @@
 <div ng-controller="ProjectController">
   <project-page>
-    <h1>Builds</h1>
-    <div style="margin-bottom: 10px;" ng-repeat="build in builds">
-      <h3>{{build.metadata.labels.buildconfig}} <span class="small">({{build.metadata.name}})</span></h3>
-      <div>Created: <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp></div>            
-      <div>Status: {{build.status}}</div>
-      <div>Build strategy: {{build.parameters.strategy.type}}</div>
-      <div ng-if="build.parameters.strategy.type == 'STI'">
-        Builder image: {{build.parameters.strategy.stiStrategy.builderImage}}
+    <div ng-controller="BuildsController">
+      <h1>Builds</h1>
+      <div ng-if="(builds | hashSize) == 0">
```

Comment:
Can we move this logic into the controllers? It seems better to have the controller indicate different states:
```
info, "Loading..."
info, "No builds to show"
warning, "Active filters are hiding all builds"
error, "Could not retrieve builds"
```
Then the view can just render the empty message and status